### DISCUSSION TITLE
Saturable tubular reabsorption

### DISCUSSION THREAD

**Discussion by MarianelaLorier** [Open-Systems-Pharmacology/Forum#1240] [Questions & Problems]
Hi everyone

I am working with a PBPK model for a drug that presents a concentration dependant renal CL because of a saturable reabsorption process (CL increases with increasing plasma concentration). How can i add this in pk sim or mobi? 

Thank you

**Comment by prvmalik** [comment]
Try an influx transporter with Michaelis-Menten kinetics on apical renal epithelium. Unsure if that would work, though.
